packages/lodestar/src/api/impl/beacon/beacon.ts

```diff
@@ -66,7 +66,7 @@ export class BeaconApi implements IBeaconApi {
 
   public async getFork(): Promise<ForkResponse> {
     const state = await this.chain.getHeadState();
-    const networkId: Uint64 = this.chain.networkId;
```

Comment:
what's networkId, do we even need it since we use forkVersion?

Reply:
nope haha, i think after your standard fork API PR, we don't even use it at all